code/game/machinery/washing_machine.dm

```diff
@@ -155,6 +159,16 @@
 					QDEL_NULL(J)
 					break
 				QDEL_NULL(J)
+			for(var/T in typesof(/obj/item/clothing/suit))
```

Comment:
i wonder how expensive it is to create every single exosuit and then delete them each time a washing machine starts
perhaps making the suit result be a variable of the crayon would be a better idea?

Reply:
it isn't